configs/config.json.example

```diff
@@ -1,39 +1,131 @@
 {
+    /************************************************************
+        NOTE: Comments are allowed now because this file is
+            passed through jsmin before reading it in!
+    *************************************************************/
+
+    // Account type: "google" or "ptc"
     "auth_service": "google",
+
+    // For google it's your email (i.e. "user@gmail.com")
+    // For ptc it's your username (i.e. "pokemaster1")
     "username": "YOUR_USERNAME",
     "password": "YOUR_PASSWORD",
+
+    // Can be coordinates or an address
     "location": "SOME_LOCATION",
+
+    // Set to true if you would like to start from the last known bot location
+    "location_cache": true,
+
+    // Your google maps API key
+    // Instructions on how to get one can be found here: https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/README.md#google-maps-api-in-development
     "gmapkey": "GOOGLE_MAPS_API_KEY",
+
+    // Specifies the max circumference that the bot can walk - best to keep it pretty low
     "max_steps": 5,
+
+    /*  modes:
+            "all"   - catch pokemon + farm pokestops
+            "poke"  - start out only catching pokemon
+            "farm"  - start out only farming pokestops
+    */
     "mode": "all",
+
+    // Walk speed - 4.16 is the max speed the bot can walk and still hatch eggs
```

Comment:
@Ryurem I wasn't the one who stated that 4.16 is the max speed for hatching eggs.  I just put it here because that is what others have said and what is documented elsewhere.  If it's wrong then we can change it.